src/main/java/team/creative/creativecore/common/config/gui/ClientSyncGuiLayer.java

```diff
@@ -112,14 +112,15 @@ public void load(CheckTree<ConfigKey>.CheckTreeEntry entry) {
             
             GuiColumn second = (GuiColumn) new GuiColumn().setExpandableX();
             row.addColumn(second);
-            String caption = translateOrDefault("config." + String.join(".", holder.path() + "." + key.content.name + ".name"), key.content.name);
-            String comment = "config." + String.join(".", holder.path()) + "." + key.content.name + ".comment";
-            if (key.content != null && key.content.get() instanceof ICreativeConfigHolder)
-                second.add(new GuiButton(caption, x -> {
-                    load(key);
-                }).setTitle(Component.literal(caption)).setTooltip(new TextBuilder().translateIfCan(comment).build()));
-            else
-                second.add(new GuiLabel(caption).setTitle(Component.literal(caption)).setTooltip(new TextBuilder().translateIfCan(comment).build()));
+            if (key.content != null) {
+                // FIXME: is normal use holder.path on a string concatenation?
+                String caption = translateOrDefault("config." + String.join(".", holder.path() + "." + key.content.name + ".name"), key.content.name);
```

Comment:
I think it should be
String caption = translateOrDefault("config." + String.join(".", holder.path()) + "." + key.content.name + ".name", key.content.name);

Reply:
holder.path() returns an String array.

Reply:
Yep, the current code is actually wrong. That's why I moved the bracket to a different place to make it work (as shown above).